lib/susedistribution.pm

```diff
@@ -384,6 +385,9 @@ sub init_consoles {
 
     if (check_var('BACKEND', 'qemu')) {
         $self->add_console('root-virtio-terminal', 'virtio-terminal', {});
+        for (my $num = 1; $num < get_var('VIRTIO_CONSOLE_NUM', 1); $num++) {
```

Comment:
I imagine we want to be flexible, but why would anyone need to add yet another console? in the case of VIRTIO_CONSOLE_NUM=3 or higher

Reply:
We have this loop in https://github.com/os-autoinst/os-autoinst/pull/1156 as well. And it would not make sense now, to not loop here, as then we would create consoles in qemu and don't have them usable in os-autoinst.